components/misc/MiddleEllipsis.tsx

```diff
@@ -1,82 +1,51 @@
-import React, { Component } from 'react';
+import { useTheme } from '@material-ui/core/styles';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
 
 
-interface TextProps {
-    text: String;
-}
+const MiddleEllipsis = (props: any) => {
```

Comment:
Use type for the props instead of `any`.

https://github.com/typescript-cheatsheets/react-typescript-cheatsheet#function-components

Reply:
Ok, updated! :)